packages/gatsby-transformer-json/README.md

```diff
@@ -14,7 +14,7 @@ points to your files.
 
 ```javascript
 // In your gatsby-config.js
-plugins: [`gatsby-transformer-json`];
+plugins: [`gatsby-transformer-json`]
```

Comment:
Actually, could You update "How to use" section to show entire example `gatsby-config.js` (for example something like in https://github.com/gatsbyjs/gatsby/tree/master/packages/gatsby-transformer-yaml - just for `json` plugin instead of `yaml`?

Reply:
Sure, although it was just a drive-by edit. ;P